src/styles/colour/index.md.njk

```diff
@@ -17,7 +17,7 @@ your service meets [level AA of the Web Content Accessibility Guidelines
 ## Main colours
 
 If you are using GOV.UK Frontend or the GOV.UK Prototype Kit, use the [Sass
-variables](https://sass-lang.com/guide#topic-2) provided rather than copying the
+variables](https://frontend.design-system.service.gov.uk/sass-api-reference/) provided rather than copying the
```

Comment:
Would it make sense to deep link to https://frontend.design-system.service.gov.uk/sass-api-reference/#colours instead?

Reply:
@36degrees It probably would - thanks!

```suggestion
variables](https://frontend.design-system.service.gov.uk/sass-api-reference/#colours) provided rather than copying the
```